### ISSUE TITLE
Compile warnings

### ISSUE THREAD

**Issue by Fakkio** [rakannimer/react-google-charts#222]
Hi, 

after the last updates to my project (I'm using [Create React apps (with Typescript)](https://github.com/wmonk/create-react-app-typescript)), I'm getting these warnings from react-google-charts using `yarn start` comand.
 
```
Compiled with warnings.

./node_modules/react-google-charts/dist/index.js
1:261-262 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

./node_modules/react-google-charts/dist/index.js
1:290-294 Critical dependency: the request of a dependency is an expression

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
```

I have no idea what it is, but let me know if I can help.

Thanks,
Fakkio.

**Comment by rakannimer** [comment]
Hey @Fakkio,

Thanks for reporting this, I was using parcel as the bundler and I couldn't manage to get it to emit umd & esm bundles.

I switched now to rollup for the build process.

The problem should be solved in `2.0.25`

Cheers !

**Comment by Fakkio** [comment]
It works! Thank you @rakannimer !
